azext_iot/operations/hub.py

```diff
@@ -398,11 +398,9 @@ def _parse_auth(parameters):
 
 
 def iot_device_module_list(cmd, device_id, hub_name=None, top=1000, resource_group_name=None, login=None):
-    query = "select * from devices.modules where devices.deviceId = '{}'".format(device_id)
```

Comment:
Be mindful when refactoring query operations to explicit 'list' endpoints. In the not too distant past, list operations had much lower upper bound than query with respect to total entities returned.